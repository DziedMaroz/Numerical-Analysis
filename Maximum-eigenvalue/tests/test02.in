3
1	2	3
3	2	4
7	8	90
1	2	3
0.000003
