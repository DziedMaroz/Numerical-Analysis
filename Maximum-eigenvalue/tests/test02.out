90.6120501191381
